apps/theme/layouts/forms/DefaultFormLayout.tsx

```diff
@@ -52,6 +52,17 @@ const DefaultFormLayout: FormLayoutComponent = ({
         const result = await submit(data);
         setLoading(false);
         if (result.error === null) {
+            const googleAnalyticsEvent = formData.triggers?.["google-analytics-event"];
```

Comment:
Trigger processing should not be in the layout. Triggers are handled internally, here:

```
packages/app-form-builder/src/components/Form/FormRender.tsx:173
```